assets/app/game_manager.rb

```diff
@@ -54,6 +54,15 @@ def get_games(params = nil)
     end
   end
 
+  def get_game(id = nil)
```

Comment:
why is there a default value here?

Reply:
For no special reason. I just thought it would make the code more robust